__test__/pages/hold/eddRequestPage.test.tsx

```diff
@@ -335,11 +331,94 @@ describe("EDD Request page", () => {
           discoveryItemResult={bibWithItems.resource.items[0]}
           patronId="123"
           isAuthenticated={true}
-          pageStatus="invalid"
+          errorStatus="invalid"
         />
       )
       expect(
-        screen.getByText(EDDPageStatusMessages.invalid.message)
+        screen.getByText(
+          "Some fields contain errors. Please correct and submit again."
+        )
+      ).toBeInTheDocument()
+    })
+  })
+  describe("EDD Request patron ineligibility messaging", () => {
+    beforeEach(async () => {
+      render(
+        <EDDRequestPage
+          discoveryBibResult={bibWithItems.resource}
+          discoveryItemResult={bibWithItems.resource.items[0]}
+          patronId="123"
+          isAuthenticated={true}
+        />
+      )
+
+      global.fetch = jest.fn().mockImplementationOnce(() =>
+        Promise.resolve({
+          status: 401,
+          json: () =>
+            Promise.resolve({
+              success: false,
+              patronEligibilityStatus: {
+                eligibility: false,
+                expired: true,
+                moneyOwed: true,
+                ptypeDisallowsHolds: true,
+                reachedHoldLimit: true,
+              },
+            }),
+        })
+      )
+
+      await fillRequiredEDDFormFields()
+
+      await fireEvent(
+        screen.getByText("Submit request"),
+        new MouseEvent("click")
+      )
+    })
+
+    it("shows an error listing ineligibility reasons when the patron is ineligibile to place holds", async () => {
+      await waitFor(() => {
+        expect(screen.getByTestId("hold-request-error")).toBeInTheDocument()
+      })
+
+      expect(
+        screen.getByText("There is a problem with your library account.", {
```

Comment:
Can you use that same pattern here with the constants for the patron  error text? Or is that not possible because the patronIneligibilityErrors include JSX elements?

Reply:
I can't use that pattern for the ineligibility reasons because of the JSX but I can for the header.